### PR TITLE
Updates to team projects via Storyblok

### PR DESCRIPTION
**Pull request by gabalafou**

### CODE REVIEW

File: apps/labs/CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
+October 17, 2024
+
+Via Storyblok:
+
+- Updates to the team projects
```
Comment (pavithraes) [inline apps/labs/CHANGELOG.md:5]:
```suggestion
- Updates to the team projects
- Move 2024 interns from `team` to `author` (remove from team page)
```

**Review by pavithraes — APPROVED** [review]
@gabalafou - I also updated interns' roles on Storyblok, shall we include that in the changelog as well? I'll approve because this looks good, and we can merge whenever. :)

**Comment by gabalafou** [comment]
I added your suggestion, will wait for CI, and if all is good, will merge

**Comment by pavithraes** [comment]
CI is green, merging :)

**Pull request merged**